src/main/java/cms/gov/madie/measure/config/MigrateSupplementalDataElementsChangeUnit.java

```diff
@@ -0,0 +1,74 @@
+package cms.gov.madie.measure.config;
+
+import java.util.List;
+
+import com.nimbusds.oauth2.sdk.util.CollectionUtils;
+
+import cms.gov.madie.measure.repositories.MeasureRepository;
+import gov.cms.madie.models.measure.Measure;
+import gov.cms.madie.models.measure.MeasureMetaData;
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@ChangeUnit(id = "migrate_supplemental_data_elements", order = "1", author = "madie_dev")
+public class MigrateSupplementalDataElementsChangeUnit {
+
+  @Setter private List<Measure> tempMeasures;
+
+  @Execution
+  public void migrateSupplementalDataElements(MeasureRepository measureRepository)
+      throws Exception {
+    log.debug("Entering migrateSupplementalDataElements()");
+
+    List<Measure> measureList = measureRepository.findAll();
+    if (CollectionUtils.isNotEmpty(measureList)) {
+      setTempMeasures(measureList);
+      measureList.stream()
+          .forEach(
+              measure -> {
+                if (CollectionUtils.isNotEmpty(measure.getSupplementalData())) {
+                  StringBuilder sb = new StringBuilder();
+                  measure
+                      .getSupplementalData()
```

Comment:
Turns out `description` can be empty. Thinking we should check for content before appending to avoid saving " -", which doesn't seem helpful.

```suggestion
                      .getSupplementalData()
                      .stream().filter(sde -> StringUtils.isNotEmpty(sde.getDescription()))
```

Reply:
@jkotanchik-SB but if we filter out the descriptions that are null, we won't concatenate the ones that only has definition?

Reply:
Yea, I think I missed something. Are we going to continue to save the selected defines to the `measure.supplementalData` List?

Reply:
@jkotanchik-SB As of now, the frontend is not touched, so I think we probably will continue to save the selected defines, until more changes are done